edx/analytics/tasks/insights/location_per_course.py

```diff
@@ -517,6 +517,7 @@ def query(self):
             FROM student_courseenrollment sce
             LEFT OUTER JOIN auth_user au on sce.user_id = au.id
             LEFT OUTER JOIN last_country_of_user uc on au.username = uc.username
+            WHERE au.username NOT LIKE 'anon__%'
```

Comment:
Does this mean LTI won;t be filtered out here?

Reply:
yes

Reply:
Discussed f2f.